### PR TITLE
issue.md changes

### PR DESCRIPTION
**Pull request by AWare**
this updates the issue template, have discussed with @SiAdcock but would appreciate any other feedback

### CODE REVIEW

**Review by SiAdcock — APPROVED** [review]

**Review by katebee — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+Hi, thanks for raising an issue with The Guardian's website.
+
+Please note that it may not be possible for us to comment on every issue publicly,
+but we will always do our utmost to resolve your problem
+and update you on our progress.
+
+We also request that you email userhelp@theguardian.com with a link to your post,
+they will be most able to respond to your concerns.
+
+Finally, any issues which just relate to editorial or commerical decisions will
```
Comment (paperboyo) [inline .github/ISSUE_TEMPLATE.md]: Typo in commercial

Comment (alfavata) [inline .github/ISSUE_TEMPLATE.md]: Typo: `commerical`

**Review by paperboyo — APPROVED** [review]

**Comment by AWare** [comment]
Thanks @alfavata

**Review by NataliaLKB — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+<!--
+
+Hi, thanks for raising an issue with The Guardian's website.
+
+Please note that it may not be possible for us to comment on every issue publicly,
+but we will always do our utmost to resolve your problem
```
Comment (nicl) [inline .github/ISSUE_TEMPLATE.md]: Nitpicking, but I would reword this section to:

`We will do our best to respond to all issues raised, but cannot guarantee to do so.`

reason being, there will be issues we cannot resolve, and others we do not want to.
